includes/multi-currency/Compatibility/WooCommerceProductAddOns.php

```diff
@@ -125,8 +125,10 @@ public function get_item_data( $addon_data, $addon, $cart_item ): array {
 				// Quantity/multiplier add on needs to be split, calculated, then multiplied by input value.
 				$price = $this->multi_currency->get_price( $addon['price'] / $addon['value'], 'product' ) * $addon['value'];
 			}
-			$price = \WC_Product_Addons_Helper::get_product_addon_price_for_display( $price, $cart_item['data'] );
-			$name .= ' (' . wc_price( $price ) . ')';
+			if ( class_exists( '\WC_Product_Addons_Helper' ) ) {
```

Comment:
This check makes sense since we were assuming the Product Addons plugin would always have the helper class when it may not.